tests/rptest/clients/kcl.py

```diff
@@ -182,6 +187,89 @@ def describe_topic(self,
 
         return self._cmd(cmd, attempts=1)
 
+    def alter_partition_reasssignments(self, topics: dict[str,
+                                                          dict[int,
+                                                               list[int]]]):
+        """
+        :param topics: the key is a topic and the value is a dict that maps partition IDs
+                       to new replica assignments
+        :return: list of KclAlterPartitionReassignmentsResponse
+        """
+        cmd = ["admin", "partas", "alter"]
+
+        for topic in topics:
+            assert len(topics[topic]) > 0
+            reassignment_str = f"{topic}:"
+            partitions = []
+            for pid in topics[topic]:
+                part_str = f"{pid}"
+                if len(topics[topic][pid]) > 0:
+                    part_str = f"{pid}->{topics[topic][pid]}"
+                    # Remove empty and [] characters
+                    part_str = part_str.replace('[', '').replace(']', '')
+                    part_str = part_str.replace(' ', '')
+                partitions.append(part_str)
+            join_partitions = ";".join(partitions)
+            reassignment_str += join_partitions
+            cmd.append(reassignment_str)
+
+        lines = self._cmd(cmd, attempts=1).splitlines()
```

Comment:
Looks like the other methods in this class just return the value of `self._cmd` i'd keep with that standard then

Reply:
> Looks like the other methods in this class just return the value of `self._cmd` i'd keep with that standard then

This is not true. See the return values for `list_offsets` and `offset_for_leader_epoch` in the `KCL` class